doc/src/explore/tutorials.md

```diff
@@ -35,45 +35,45 @@ DB4C7667636471AFF396B900EB7B63FACAF629B6
 A6BBB1930E01495EE93CE912EA01C29695E07890
 ```
 Note that since these addresses are random generated, they will be different from what you see. We are going to need three addresses to play TicTacToe. Let's pick the first three addresses. Let's call them ADMIN, PLAYER_X and PLAYER_O.
-Since we will be using these addresses and gas objects repeatedly in the rest of this tutorial, let's make them environment variables so that we don't have to retype them every time:
+Since we will be using these addresses and Sui (more generally known as _gas_) objects repeatedly in the rest of this tutorial, let's make them environment variables so that we don't have to retype them every time:
 ```
 export ADMIN=ECF53CE22D1B2FB588573924057E9ADDAD1D8385
 export PLAYER_X=7B61DA6AACED7F28C1187D998955F10464BEAE55
 export PLAYER_O=251CF224B6BA3A019D04B6041357C20490F7A322
 ```
 
-For each of these addresses, let's discover their gas objects for each account address:
+For each of these addresses, let's discover their Sui objects for each account address:
 ```
-$ wallet --no-shell gas --address $ADMIN
-                Object ID                 |  Version   |  Gas Value
+$ wallet --no-shell sui --address $ADMIN
+                Object ID                 |  Version   |  Sui Value
 ----------------------------------------------------------------------
  38B89FE9F4A4823F1406938E87A8767CBD7F0B93 |     0      |   100000
  4790500A28AB5B4F9A3988E2A5E201D56996CBB0 |     0      |   100000
  6AB7D15F41B28FF1EBF6D32499214BBD9035D1EB |     0      |   100000
  800F2704E22637A036C4325B539D711BB83CA6C2 |     0      |   100000
  D2F52301D5343DD2C1FA076401BC6283C3E4AA34 |     0      |   100000
-$ wallet --no-shell gas --address $PLAYER_X
-                Object ID                 |  Version   |  Gas Value
+$ wallet --no-shell sui --address $PLAYER_X
+                Object ID                 |  Version   |  Sui Value
 ----------------------------------------------------------------------
  6F675038CAA48184707DBBE95ACFBA2030E87CD8 |     0      |   100000
  80C91F0B31EFBC1C7BF639A531301AAF3A1D3AB6 |     0      |   100000
  9FED1FC3D21F284DC53DE87C0E19718971D96D8C |     0      |   100000
  E293F935F015C23216867442DB4E712518E7CAB7 |     0      |   100000
  F19384C06AE538F9C3C9D9762002B4DAEA49FE3A |     0      |   100000
-$ wallet --no-shell gas --address $PLAYER_O
```

Comment:
These commands have not been updated for sure. Nor should they necessarily, I think (please see my other comment).

More generally, wallet outputs have not been updated to the new terminology (again, I don't think it's needed in may, possibly all, cases).